### ISSUE TITLE
Implement layout of feed page

### ISSUE THREAD

**Issue by jordansimsmith** [softeng-701-group-5/softeng-701-assignment-1#40]
**User Story**

As a frontend developer, I'd like to implement the layout of the feed page, so that I have a base to make feed requests and render feed data.

**Acceptance Criteria**

- Material UI components used
- Layout implements the wireframe on the wiki
- Unit tests added for the page

**Notes**
Calling the feed endpoint and rendering the cards are out of scope of this issue. However, ensure that the design would allow for the addition of these features in the future with minimal effort.

**Comment by AlexanderTheGrape** [comment]
Looks good, I'll approve this issue. I believe Kimberley was going to upload her wireframe to the wiki, so that'll need to be completed first

**Comment by Kalashnikkov** [comment]
@AlexanderTheGrape Let's try adding the Approved label when there are 2 or more 👍 's! But LGTM, nice work guys

**Comment by AlexanderTheGrape** [comment]
Ok gotcha, didn't know we had that rule for introducing new issues. Are we still doing 3 approvals for PR's?

**Comment by AlexanderTheGrape** [comment]
@Kalashnikkov

**Comment by Matteas-Eden** [comment]
Due to some permission issues with GitHub that couldn't be resolved, we decided for PRs that we would only merge after 3 approvals. Although, technically, only 2 approvals are needed (from mergers) before it is able to be merged (as far as GitHub is concerned anyway). The third approval is only enforced by us, not by GitHub. 
Hope that clears it up :)

**Comment by jordansimsmith** [comment]
> Looks good, I'll approve this issue. I believe Kimberley was going to upload her card and/or wireframe drawings to the wiki, so that may need to be completed first

https://github.com/softeng-701-group-5/softeng-701-assignment-1/wiki/Frontend-Meeting

Wireframe here.
